src/cli/tls_http_server.cpp

```diff
@@ -513,7 +513,7 @@ class TLS_HTTP_Server final : public Command
          {
          if(size_t t = get_arg_sz("threads"))
             return t;
-         if(size_t t = std::thread::hardware_concurrency())
+         if(size_t t = get_cpu_total())
```

Comment:
Should be `Botan::OS::` prefixed. Also, seems better to use `get_cpu_available`? Since if a CPU is offline it doesn't make sense to schedule an extra thread for it.

Reply:
yes it s clearer prefixed even though it compiled. Your last point is valid too.